pkg/metrics/cluster/node.go

```diff
@@ -150,10 +148,11 @@ func newRing(cfg ring.Config, name, key string, reg prometheus.Registerer, log l
 		kv.RegistererWithKVName(reg, name+"-ring"),
 		log,
 	)
+
 	if err != nil {
 		return nil, err
 	}
-	return ring.NewWithStoreClientAndStrategy(cfg, name, key, store, ring.NewIgnoreUnhealthyInstancesReplicationStrategy())
+	return ring.NewWithStoreClientAndStrategy(cfg, name, key, store, ring.NewIgnoreUnhealthyInstancesReplicationStrategy(), reg, log)
```

Comment:
You'll want to wrap the registerer here as well. Since we removed the cortex_ prefix on the ring metrics in dskit you'll want to wrap with the cortex_ prefix here to match the previous cortex metrics.